package.json

```diff
@@ -14,46 +14,54 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/blimmer/ember-cli-server-variables.git"
+    "url": "githttps://github.com/blimmer/ember-cli-server-variables.git"
```

Comment:
Is this the correct format? I think it need to be `git+https` or just `https`

Reply:
Sorry - not sure how that got in there - my editor I think.  I put the `+` back in and fixed it up.